igcollect/linux_disklatency.py

```diff
@@ -7,11 +7,13 @@
 from argparse import ArgumentParser
 from subprocess import check_output
 from time import time
-import sys
```

Comment:
I think the `sys` module was present to allow using `sys.exit` instead of `exit`. All tough it wasn't used below we should use it.

See [this](https://stackoverflow.com/questions/6501121/difference-between-exit-and-sys-exit-in-python) StackOverflow post for a good explanation.